recipe/meta.yaml

```diff
@@ -22,6 +22,8 @@ requirements:
     - pip
     - python
     - pdm-pep517 >=0.3.0
+    - setuptools
+    - wheel
   run:
     - cachecontrol >=0.12.11
     - certifi >=2022.6.15
```

Comment:
I don't see pip listed: https://github.com/pdm-project/pdm/blob/2.2.1/pyproject.toml#L10
importlib-metadata is for py<310: https://github.com/pdm-project/pdm/blob/2.2.1/pyproject.toml#L29